js/index.js

```diff
@@ -0,0 +1,86 @@
+import { Book } from '../modules/Book.js';
+import { DateTime } from '../modules/luxon.js';
+
+const addBtn = document.getElementById('addBtn');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+
+function addBook(e) {
```

Comment:
- [ ] Please, consider to use arrow function syntax 👍

Reply:
Done

Reply:
Done